packages/server/src/app/db/sections/sections.module.ts

```diff
@@ -0,0 +1,48 @@
+import { Module } from '@nestjs/common';
+import { MongooseModule } from '@nestjs/mongoose';
+import { HookNextFunction } from 'mongoose';
+import { generate } from 'shortid';
+
+import { sanitizeHtml, stripAllHtml } from '@pulp-fiction/html_sanitizer';
+import { SectionsDocument, SectionsSchema } from './sections.schema';
+import { SectionsService } from './sections.service';
+import { countPlaintextWords, countQuillWords } from '@pulp-fiction/word_counter';
+
+@Module({
+  imports: [
+    MongooseModule.forFeatureAsync([
+      {
+        name: 'Section',
+        useFactory: () => {
+          const schema = SectionsSchema;
+          schema.pre<SectionsDocument>('save', async function (next: HookNextFunction) {
+            this.set('_id', generate());
```

Comment:
this shouldn't be necessary, as it's already done in the `@Prop()` on the schema